akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -513,14 +290,14 @@ final class GraphInterpreter(
     var i = 0
     while (i < logics.length) {
       val logic = logics(i)
-      logic.stageId = i
       logic.interpreter = this
       try {
         logic.beforePreStart()
         logic.preStart()
       } catch {
         case NonFatal(e) ⇒
-          log.error(e, "Error during preStart in [{}]", assembly.stages(logic.stageId))
+          e.printStackTrace()
```

Comment:
This should be fixed. Probably logics should have a var pointing to their stage. Not sure if stageId is needed anymore, I think it can be just removed. See https://github.com/akka/akka/issues/22433